packages/lib-core/src/utils/__tests__/logger.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { consoleLogger } from '../logger';
+
+describe('consoleLogger', () => {
+  beforeEach(() => {
+    jest.resetModules();
+  });
+
+  test('should have all console loggers', () => {
+    expect(consoleLogger.warn).toBeDefined();
+    expect(consoleLogger.error).toBeDefined();
+    expect(consoleLogger.info).toBeDefined();
+  });
+
+  test('should have all console loggers except info', () => {
+    const consoleSpy = jest.spyOn(console, 'info');
+    process.env.NODE_ENV = 'production';
+    expect(consoleLogger.warn).toBeDefined();
+    expect(consoleLogger.error).toBeDefined();
+    expect(consoleLogger.info).toBeDefined();
+    consoleLogger.info('foo');
+    expect(consoleSpy).not.toBeCalled();
```

Comment:
This is fair, I guess we don't care what the impl is... as long as it doesn't blow up and doesn't call `console.info`